trin-history/src/storage.rs

```diff
@@ -29,6 +29,7 @@ pub struct HistoryStorage {
     node_id: NodeId,
     node_data_dir: PathBuf,
     storage_capacity_in_bytes: u64,
+    network_content_storage_used: u64,
```

Comment:
I don't love this attribute name... Imo, `network` is somewhat redundant, since we're already inside `HistoryStorage` & imo that implies that all content that we're dealing with is already from the history network. I also think there's a better word than `_used` that we can "use" (lol the irony).

Reply:
Maybe... if you agree.... `storage_capacity_in_bytes` also feels somewhat redundant / probably made more sense when we had a single `Storage` struct. What about...
`storage_capacity_in_bytes` -> `bytes_capacity`
`network_content_storage_used` -> `bytes_occupied`

Reply:
I like `storage_capacity_in_bytes`/`storage_allocated_in_bytes` as it makes it clear what the variable is without relying on knowing what struct you are in. 

`bytes_occupied` of what? having `storage_capacity` prefix makes it more readable and less ambigious. Expessially for people less familar with the codebase.